src/components/results/securityanalysis/security-analysis.type.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { ColDef } from 'ag-grid-community/dist/lib/entities/colDef';
+
+export interface LimitViolation {
+    subjectId: string;
+    limitType: string;
+    limit: number;
+    value: number;
+    loading: number | undefined;
+}
+export interface LimitViolationFromBack extends LimitViolation {
+    limitReduction: number;
+    limitName?: string;
+    acceptableDuration?: number;
+    side?: string;
+}
+export interface LimitViolationsResult {
+    limitViolations: LimitViolationFromBack[];
+}
```

Comment:
```suggestion
export interface LimitViolationsResultFromBack {
    limitViolations: LimitViolationFromBack[];
}
```

Reply:
Done, even if I don't see why is it helpful

Reply:
I might be wrong for this one
since this interface is based on LimitViolationFromBack, I thought it should be named (...)FromBack as well

Reply:
Done